pkg/storage/aws/s3.go

```diff
@@ -0,0 +1,48 @@
+package aws
+
+import (
+	"context"
+
+	"errors"
+
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	// "github.com/pkg/errors" don't need stack trace, not inside velero-plugin
+)
+
+func BucketRegionIsDiscoverable(bucket string) bool {
+	_, err := GetBucketRegion(bucket)
+	return err == nil
+}
+
+// GetBucketRegion returns the AWS region that a bucket is in, or an error
+// if the region cannot be determined.
+// copied from https://github.com/openshift/openshift-velero-plugin/pull/223/files#diff-da482ef606b3938b09ae46990a60eb0ad49ebfb4885eb1af327d90f215bf58b1
```

Comment:
here we can't import plugin because of cyclic import?

Reply:
but plugin imports oadp, so we could tell it to import from here the function, and avoid duplicating code, no?

Reply:
Yeah importing plugin go package would bump k8s

Reply:
Which breaks @shubham-pampattiwar knows the pain